lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -122,10 +122,14 @@ define([
          * Shows error message.
          */
         showErrorMessage: function (message) {
-            this.$('.wrapper-msg')
-                .removeClass('is-hidden')
-                .find('.msg-content .copy').html(message);
+            var $wrapper = this.$('.wrapper-msg');
+            $wrapper.removeClass('is-hidden');
+
+            if (!(message instanceof HtmlUtils.HtmlSnippet)) {
```

Comment:
I think this logic makes this method dangerous, because the caller can pass any arbitrary string and this logic will then declare it to be safe HTML. My preference would be to rename this as ``showErrorMessageHtml`` and to have the code use ``HtmlUtils.ensureHtml`` on the parameter. It is then up to the callers to provide safely escaped HTML snippets in the first place.

In general, I wouldn't expect methods outside of the UI Toolkit to perform ``instanceof`` checks on HTML. The intention is that this is an implementation detail of the UI Toolkit to provide safe HTML.